addon/templates/components/offering-form.hbs

```diff
@@ -1,74 +1,78 @@
 {{#if loaded}}
   <div class="toggle-offering-calendar">
     {{toggle-buttons
-      firstOptionSelected=(not showOfferingCalendar)
       firstLabel=(t "general.hideCalendar")
+      firstOptionSelected=(not showOfferingCalendar)
       secondLabel=(t "general.showCalendar")
-      toggle=(toggle "showOfferingCalendar" this)
-    }}
+      toggle=(toggle "showOfferingCalendar" this)}}
   </div>
+
   {{#if showOfferingCalendar}}
     {{offering-calendar
-      startDate=startDate
       endDate=endDate
       learnerGroups=learnerGroups
-      session=session
-    }}
+      startDate=startDate
+      session=session}}
   {{/if}}
 
   <div class="form">
     <div class="item start-date">
       <label>{{t "general.startDate"}}:</label>
       {{pikaday-input
-        value=startDate
-        onSelection=(action "updateStartDate")
         format="M/D/YYYY"
-      }}
+        value=startDate
+        onSelection=(action "updateStartDate")}}
     </div>
 
     <div class="item start-time">
-      <label>{{t "general.startTime"}}: <span class="timezone">({{browser-timezone}})</span></label>
+      <label>
+        {{t "general.startTime"}}: <span class="timezone">({{browser-timezone}})</span>
+      </label>
       {{time-picker date=startDate action=(action "updateStartTime")}}
     </div>
 
     <div class="item offering-duration">
       <label>{{t "general.duration"}}:</label>
       <div class="duration-inputs">
         <div class="hours">
-          <input
-            type="text"
-            value={{durationHours}}
-            oninput={{action (perform updateDurationHours) value="target.value"}}
-            onkeyup={{action "addErrorDisplayFor" "durationHours"}}
-            class={{if
-              (and (v-get this "durationHours" "isInvalid") (is-in showErrorsFor "durationHours"))
-              "has-error"
-            }}
-          >
-          {{#if
-            (and (v-get this "durationHours" "isInvalid") (is-in showErrorsFor "durationHours"))
-          }}
-            <span class="validation-error-message">{{v-get this "durationHours" "message"}}</span>
-          {{/if}}
-          <label>{{t "general.hours"}}</label>
+          <div class="hours__container">
```

Comment:
Prefer dash to double underscore so this would be `hours-container`.

Reply:
👍